### PR TITLE
Connect Database 

### PR DESCRIPTION
**Pull request by RavinduLakshitha**
I connected to the database. Check it

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -1,9 +1,19 @@
-const express = require('express');
+const express = require("express");
+const mongoose = require("mongoose");
 
 const app = express();
 
-const port = 3000;
+const PORT = 3000;
+const DB_URL =
+  "mongodb+srv://ravi123:Abc123@cluster0.fbt0o.mongodb.net/internX";
```
Comment (bihanviranga) [inline server.js]: Let's move username/password to a `.env` file.

File: templates/login.hbs [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>login</title>
+  </head>
+  <body>
+    <form action="/login" method="post"></form>
+  </body>
+</html>
```
Comment (bihanviranga) [inline templates/login.hbs]: We don't need templating for this, let's just use a REST API.

**Review by bihanviranga — CHANGES_REQUESTED** [review]
Added some comments

File: server.js (hunk 1 above)
Comment (RavinduLakshitha) [reply server.js]: I changed it. it's moved to the.env file

**Review by Lakshi99 — COMMENTED** [review]
Great! keep it up

**Pull request merged**